content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+arises you can easily roll the change back by deleting the rolebinding. Just make sure the
+pre-existing PodSecurityPolicies are still in place!
+
+```sh
+# Undo PodSecurityPolicy disablement.
+kubectl delete -n $NAMESPACE rolebinding disable-psp
+```
+
+## 4. Review namespace creation processes
+
+Now that existing namespaces have been updated to enforce Pod Security Admission, you should ensure
+that your processes and/or policies for creating new namespaces are updated to ensure that an
+appropriate Pod Security profile is applied to new namespaces.
+
+You can also statically configure the Pod Security admission controller to set a default enforce,
+audit, and/or warn level for unlabeled namespaces. See [Configure the Admission
+Controller](docs/tasks/configure-pod-container/enforce-standards-admission-controller/#configure-the-admission-controller)
+for more information.
+
+## 5. Disable PodSecurityPolicy
+
+Finally, you're ready to disable PodSecurityPolicy. To do so, you will need to modify the admission
+configuration of the API server: [How do I turn off an admission
+controller?](/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-off-an-admission-controller).
+
+To verify that the PodSecurityPolicy admission controller is no longer enabled, you can manually run
+a test by impersonating a user without access to any PodSecurityPolicies (see the [PodSecurityPolicy
+example](/docs/concepts/policy/pod-security-policy/#example)), or by verifying in the API server
+logs. At startup, the API server outputs log lines listing the loaded admission controller plugins:
+
+```
+I0218 00:59:44.903329      13 plugins.go:158] Loaded 16 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,TaintNodesByCondition,Priority,DefaultTolerationSeconds,ExtendedResourceToleration,PersistentVolumeLabel,DefaultStorageClass,StorageObjectInUseProtection,RuntimeClass,DefaultIngressClass,MutatingAdmissionWebhook.
+I0218 00:59:44.903350      13 plugins.go:161] Loaded 14 validating admission controller(s) successfully in the following order: LimitRanger,ServiceAccount,PodSecurity,Priority,PersistentVolumeClaimResize,RuntimeClass,CertificateApproval,CertificateSigning,CertificateSubjectRestriction,DenyServiceExternalIPs,ValidatingAdmissionWebhook,ResourceQuota.
+```
+
+You should see `PodSecurity` (in the validating admission controllers), and neither list should
+contain `PodSecurityPolicy`.
+
+Once you are certain the PSP admission controller is disabled (and after sufficient soak time to be
+confident you won't need to roll back), you are free to delete your PodSecurityPolicies and any
+associated Roles, ClusterRoles, RoleBindings and ClusterRoleBindings (just make sure they don't
+grant any other unrelated permissions).
+
+## Appendix: Pod Security Policies to Pod Security Standards {#psp-fields}
```

Comment:
I would put this section / table as a standalone page in the Reference section, and hyperlink to it.

(Appendices to tasks can be harder to find than standalone reference pages, because maybe the task you're performing is “find out what Pod Security Standard I should use for this namespace, the vendor's docs only mention a PodSecurityPolicy” and we don't have a task page for that).

Reply:
Sounds good, I'll follow up with this.